### PR TITLE
Login fix

### PR DESCRIPTION
**Pull request by HarryHenryGebel**
Store userId in local storage on login, remove it on logout, reload page on login / logout

### CODE REVIEW

File: src/components/Nav/DashboardNav.jsx [hunk 1]
```diff
@@ -85,7 +85,13 @@ function DashboardNav(props) {
           ))}
 
           <DropdownItem divider />
-          <DropdownItem onClick={() => localStorage.removeItem("token")}>
+          <DropdownItem
+            onClick={() => {
+              localStorage.removeItem("token");
```
Comment (Jahteo) [inline src/components/Nav/DashboardNav.jsx:90]: I understand what you removed, but not why the second or third line are needed. Just above my pay-grade so far.

File: src/components/Nav/LoginForm.jsx [hunk 2]
```diff
@@ -61,9 +59,10 @@ export default function LoginForm() {
         }
       )
       .then((res) => {
-        console.log(res);
-        localStorage.setItem("token", res.data.access_token);
-        history.push("/");
+        storeLoginInformation(res.data.access_token, username);
+        // kludgily refresh screen
+        // history.push("/");
+        window.open("/", "_self");
```
Comment (Jahteo) [inline src/components/Nav/LoginForm.jsx:65]: history.push() was already the limit of my knowledge. window.open is next level.

File: src/components/Nav/LoginForm.jsx [hunk 3]
```diff
@@ -41,18 +40,17 @@ export default function LoginForm() {
...
       password: "",
     });
+
+    const username = formState.username.trim(),
+      password = formState.password.trim();
     axiosWithAuth()
       .post(
         "/login",
-        `grant_type=password&username=${user.username}&password=${user.password}`,
+        `grant_type=password&username=${username}&password=${password}`,
```
Comment (Jahteo) [inline src/components/Nav/LoginForm.jsx:53]: I understand these as cleaning up already functional code, nicely done.

File: src/components/Nav/LoginForm.jsx [hunk 4]
```diff
@@ -103,44 +102,42 @@ export default function LoginForm() {
...
-                  Username
-                </Label>
-                <Input
-                  type="username"
-                  name="username"
-                  id="username"
-                  value={formState.username}
-                  onChange={inputChange}
-                  placeholder="That thing you called yourself"
+            <FormGroup className="mb-2 mr-sm-2 mb-sm-0">
```
Comment (Jahteo) [inline src/components/Nav/LoginForm.jsx:105]: Reactstrap is weird. As long as this didn't break the formatting and functionality that they baked into Form. I know moving seemingly redundant bits ruined other things I worked on.

File: src/components/Nav/RegistrationForm.jsx [hunk 5]
```diff
@@ -33,7 +32,7 @@ export default function RegistrationForm() {
     password: "",
     primaryemail: "",
     iimageurl: "",
-    terms: ""
+    terms: "",
```
Comment (Jahteo) [inline src/components/Nav/RegistrationForm.jsx:35]: interesting. last trailing comma in an obj is purely aesthetic choice, right? was it just prettify that did this or is there another reason why like edge cases or performance? Purely curious.

File: src/components/Nav/RegistrationForm.jsx [hunk 6]
```diff
@@ -97,80 +95,78 @@ export default function RegistrationForm() {
...
-                <FormText>
-                  After logging in, you may edit more profile options.
-                </FormText>
-              </FormGroup>
-            </Form>
+              </Label>
+              <FormText>
+                After logging in, you may edit more profile options.
+              </FormText>
+            </FormGroup>
```
Comment (Jahteo) [inline src/components/Nav/RegistrationForm.jsx:169]: I think all that changed here is the removal of the seemingly redundant <Form> <Form/> again. git is showing it weirdly epic though, so not sure if i'm missing anything.

File: src/utils/index.js [hunk 7]
```diff
@@ -1,8 +1,15 @@
-import axiosWithAuth from "./axiosWithAuth";
-import useForm from './useForm'
-import useFinder from './useFinder'
-
-export { axiosWithAuth, useForm, useFinder };
-
-
-
+import {
```
Comment (Jahteo) [inline src/utils/index.js:1]: understand what you did and how, but no clue what it changed/impacted

File: src/components/Nav/DashboardNav.jsx (hunk 1 above)
Comment (HarryHenryGebel) [reply src/components/Nav/DashboardNav.jsx:90]: I didn't change anything here, prettier did that.

File: src/components/Nav/RegistrationForm.jsx (hunk 5 above)
Comment (HarryHenryGebel) [reply src/components/Nav/RegistrationForm.jsx:35]: This is another one I didn't touch, prettier again.

File: src/utils/index.js (hunk 7 above)
Comment (HarryHenryGebel) [reply src/utils/index.js:1]: Prettier.

**Pull request closed without merging**